### ISSUE TITLE
lack of proper documentation

### ISSUE THREAD

**Issue by Foadsf** [nschloe/flow#8]
I ended up in this repository googling FEniCS+compressible. It would be great if you could add some proper documentations, examples etc. Thanks.

**Comment by nschloe** [comment]
Yeah, completely right. It'll probably be a long while before I get to that though. :/

**Comment by nschloe** [comment]
In the meantime, I'd recommend checking out Jan Blechta's work at https://github.com/blechta/fenapack.
